platform/main-glue.roc

```diff
@@ -0,0 +1,28 @@
+platform "glue"
+    requires {} { main : GlueTypes }
+    exposes []
+    packages {}
+    imports [
+        InternalCommand,
+        InternalFile,
+        InternalTcp,
+    ]
+    provides [mainForHost]
+
+GlueTypes : [
+    A InternalCommand.Command,
+    B InternalCommand.Output,
+    C InternalCommand.CommandErr,
+    D InternalFile.ReadErr, 
+    E InternalFile.WriteErr,
+    F InternalTcp.Stream,
+    G InternalTcp.ConnectErr,
+    H InternalTcp.StreamErr,
+    I InternalTcp.ConnectResult,
+    J InternalTcp.WriteResult,
+    K InternalTcp.ReadResult,
+    L InternalTcp.ReadExactlyResult
+]
```

Comment:
We can add types here so they will be included in the generated glue. Generation fails when running glue on platform/main.roc.